lc3/README.rst

```diff
@@ -0,0 +1,186 @@
+.. _lc3:
```

Comment:
File needs to be added here: https://github.com/nrfconnect/sdk-nrfxlib/blob/main/README.rst